ttnn/cpp/pybind11/pytensor.cpp

```diff
@@ -525,7 +546,7 @@ std::variant<OwnedBuffer, BorrowedBuffer> get_host_buffer_from_tensor(const Tens
                                 : unpack_bfp4_tiles_into_float_vec(
                                       uint32_data, /*row_major_output=*/false, /*is_exp_a=*/false, tile);
```

Comment:
Not related to this PR, but just wondering, if we can optimize things a bit by setting `row_major_output = true`?

Reply:
Probably, but i don't know how well that path is tested.